layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,42 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode 
+                      f 
+                      dash-functional
```

Comment:
`dash-functional` is just another file in `dash` package so if `dash` is installed, it should be `require`able.